infra/gcb/build_project.py

```diff
@@ -21,6 +21,8 @@
 from oauth2client.service_account import ServiceAccountCredentials
 from googleapiclient.discovery import build
 
+import build_helper
```

Comment:
super nit: `helper` doesn't sound the best to me as a utility module. Maybe `build_lib` or something similar would be better, or at least make this plural (build_helper**s**).

Reply:
thanks, renamed to `build_lib`!